docs/pages/plugins-manifest.md

```diff
@@ -81,55 +81,78 @@ Defines the minimum version of Sketch required to run the plugin. This string mu
+  // ... plugin code to select all
+}
+```
+
+| Command field | Description                                                                                                                                                                                                                                                                                                                                                                                                      |
+| ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `identifier`  | Defines a unique identifier of the command within the plugin bundle.                                                                                                                                                                                                                                                                                                                                             |
+| `name`        | Provides the name of the command which is used within the _Plugins_ menu.                                                                                                                                                                                                                                                                                                                                        |
+| `shortcut`    | Provides a default keyboard shortcut for the command, e.g. `ctrl shift t`.                                                                                                                                                                                                                                                                                                                                       |
+| `script`      | Specifies the relative path within the plugin bundle's `Sketch` folder to the script implementing the command.                                                                                                                                                                                                                                                                                                   |
+| `handler`     | Specifies the name of function to be called with the command. The function should be declared at the top level of the script, and accepts a single `context` parameter which contains information such as the current document and selection.<br/><br/>If using the `skpm` tool you may `export` the function instead of declaring it, or omit this field entirely and declare the handler with `export default` |
```

Comment:
if you are not using skpm and omit the field, you need to declare an `onRun` function

Reply:
If vanilla is default it would need to be worded the other way round:

If you're using `skpm` you don't need to declare an `onRun` function.